sky/optimizer.py

```diff
@@ -1330,7 +1330,6 @@ def _fill_in_launchable_resources(
                                 f'{colorama.Style.RESET_ALL}')
                 for cloud, hint in hints.items():
                     logger.info(f'{repr(cloud)}: {hint}')
-            else:
```

Comment:
IMHO, the CPU/memory suggestion should only be shown when there is no fuzzy_candidates, otherwise we'd suggest using the fuzzy one.